tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -282,3 +287,18 @@ def run_fio_on_pvcs(self, pvc_size):
             performance_lib.write_fio_on_pod(objs, file_size_mb_str)
 
         return total_files_size
+
+    def test_bulk_clone_performance_results(self):
+        """
+        This is not a test - it is only check that previous test ran and finish as expected
+        and reporting the full results (links in the ES) of previous tests (3)
+        """
+        self.number_of_tests = 3
```

Comment:
the interface_iterate have only 2 values, so, the test will run only twice, which mean this value should be `self.number_of_tests = 2`

Reply:
Updated the code